src/components/ProgressCircle/styles.scss

```diff
@@ -8,11 +9,11 @@
 .circle-fill {
   position: absolute;
   left: 0;
-  color: $tw-clr-base-semantic-productive;
+  color: $tw-clr-icn-brand;
```

Comment:
the mappings for these tokens are here https://docs.google.com/spreadsheets/d/1ECFEO33DP0GgesPagynY5MIIx_4lFn3rJh_PWjcfqLI/edit#gid=1168800656
(see Jira https://terraformation.atlassian.net/browse/SW-2005)